doc/api/buffer.md

```diff
@@ -508,7 +508,7 @@ added: v5.10.0
 -->
 
 * `size` {Integer} The desired length of the new `Buffer`
-* `fill` {string | Buffer | Integer} A value to pre-fill the new `Buffer` with.
+* `fill` {string|Buffer|Integer} A value to pre-fill the new `Buffer` with.
```

Comment:
Yeah, I guess it's better to keep it as-is. Per our rule discussed in the last PR, everything except primit should be uppercased.